README.md

```diff
@@ -38,11 +38,11 @@ You will also see any lint errors in the console.
 
 ## Live Demo Link
 
-### [Live Demo]
+### [Live Demo](https://christabelle143-78cc21.netlify.app/)
 
 ## Loom Video Link
 
-### [Loom]
+### [Loom](https://www.loom.com/share/e826ba201e4146508c393a459515834c)
```

Comment:
Good job with project presentation, however, you missed some steps in your video presentation. Your video presentation only showed functional aspect of your app and failed to highlight any portion of your code. 
- [ ] Kindly ensure that the video contains a highlight about a piece of the code that you find interesting or that you were proud of making.